commands/outliers.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+const psql = require('../lib/psql')
+
+function * ensurePGStatStatement (db) {
+  let query = `
+SELECT exists(
+  SELECT 1 FROM pg_extension e LEFT JOIN pg_namespace n ON n.oid = e.extnamespace
+  WHERE e.extname='pg_stat_statements' AND n.nspname = 'public'
+) AS available`
+  let output = yield psql.exec(db, query)
+
+  if (!output.includes('t')) {
+    throw new Error(`pg_stat_statements extension need to be installed in the public schema first.
+    This extension is not available on hobby tier Postgres. You can install it by running:
```

Comment:
actually this may be confusing. it's more like "this extension is not installed on hobby tier Postgres by default (but you can install if you like)", I think

Reply:
So I can just remove any mention to hobby then?